start-test-solr.sh

```diff
@@ -1,11 +1,13 @@
-#!/bin/sh
+#!/bin/bash
 
 set -e
 
 SOLR_VERSION=4.7.2
 
-export SOLR_ARCHIVE="${SOLR_VERSION}.tgz"
+export SOLR_ARCHIVE="solr-${SOLR_VERSION}.tgz"
+export SOLR_DIR="solr-${SOLR_VERSION}"
```

Comment:
What do you think about changing this to make everything happen inside a directory like `test-solr-server`? That way we could have a single pattern for the .gitignore, prefix everything in here, etc.